swift-sdk/Internal/InAppPresenter.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Created by Jay Kim on 4/21/20.
+//  Copyright © 2020 Iterable. All rights reserved.
+//
+
+import UIKit
+
+class InAppPresenter {
+    var topVC: UIViewController
+    var htmlMessageVC: IterableHtmlMessageViewController
+    var delayTimer: Timer?
+    
+    init(topViewController: UIViewController, htmlMessageViewController: IterableHtmlMessageViewController) {
+        topVC = topViewController
+        htmlMessageVC = htmlMessageViewController
+        
+        htmlMessageVC.presenter = self
+    }
+    
+    func show() {
+        if #available(iOS 10.0, *) {
+            delayTimer = Timer(timeInterval: 0.15, repeats: false) { _ in
```

Comment:
Is 0.15 the ideal wait time for us? Did you test different values for this setting?

Reply:
We discussed on using 0.15 since it's around the edge of perception of what would be "too late". It can be tuned further if needed, but it's felt fine (e.g. imperceptible) in my testing thus far.